src/psyclone/dynamo0p3_builtins.py

```diff
@@ -56,9 +57,13 @@
 VALID_BUILTIN_ARG_TYPES = LFRicArgDescriptor.VALID_FIELD_NAMES + \
     LFRicArgDescriptor.VALID_SCALAR_NAMES
 
-# The data types of field arguments that are valid for built-in
-# kernels in the LFRic API (only "gh_real")
-VALID_BUILTIN_FIELD_DATA_TYPES = ["gh_real"]
+# The data types of field arguments that are valid for built-in kernels
+# in the LFRic API (also valid LFRicArgDescriptor field data types)
+VALID_BUILTIN_FIELD_DATA_TYPES = LFRicArgDescriptor.VALID_FIELD_DATA_TYPES
```

Comment:
I don't think there's any point having these `VALID_BUILTIN_...` lists now as they are identical to the LFRicArgDescriptor lists.

Reply:
Removed and now using the ones from `LFRicArgDescriptor`.